includes/express-checkout/class-wc-payments-express-checkout-button-helper.php

```diff
@@ -117,6 +117,7 @@ public function build_display_items( $itemized_display_items = false ) {
 		if ( WC()->cart->needs_shipping() ) {
 			$shipping_tax = $this->cart_prices_include_tax() ? WC()->cart->shipping_tax_total : 0;
 			$items[]      = [
+				'key'    => 'total_shipping',
```

Comment:
I am mainly curious but will we need to add the same key in the following line:
* [class-wc-payments-express-checkout-button-helper.php#L726](https://github.com/Automattic/woocommerce-payments/blob/2b912070adcb456f076c088c1b46feb29c523990/includes/express-checkout/class-wc-payments-express-checkout-button-helper.php#L726).